p2p/libp2p/netMessenger.go

```diff
@@ -208,6 +215,14 @@ func createMessenger(
 				"cross shard", len(peersInfo.CrossShardPeers),
 				"unknown", len(peersInfo.UnknownPeers),
 			)
+
+			connsPerSec := int64(conns) / int64(durationBetweenPeersPrints/time.Second)
```

Comment:
please rename `durationBetweenPeersPrints` to `secondsBetweenPeerPrints` and make it an int which holds the number of seconds. And at line 205 multiply with `time.Second`

Reply:
done